index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Video Project</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <script src='main.js'></script>
+</head>
+
+<body>
+    <div class="topnav">
+        <a href="#">First</a>
+        <a href="#">Second</a>
+        <a href="#">Third</a>
+    </div>
+    <section class="content">
+         <div>
```

Comment:
Kindly add some of the elements below the video – the title, description and number of views